chrome/js/background.js

```diff
@@ -13,4 +12,13 @@ chrome.runtime.onInstalled.addListener(function () {
       }
     ]);
   });
+  chrome.contextMenus.removeAll();
```

Comment:
Why is this removeAll needed? Could this potentially interfere with other chrome extensions?

Reply:
Should only remove things related to our own extension: https://developer.chrome.com/apps/contextMenus#method-removeAll

But we shouldn't need it, just the few examples i saw all included it. Will update after i test

Reply:
Kk once you do that I think this looks good to go so will throw you that stamp

Reply:
looks like safer to leave it in: https://stackoverflow.com/questions/38190701/why-does-chrome-contextmenus-create-multiple-entries